frontend/src/scene/payment/state.js

```diff
@@ -0,0 +1,35 @@
+import { observable, decorate, action } from 'mobx';
+import { getPaymentDetails } from '../../apis';
+
+class PaymentFormState {
+    order_number = '';
+    submitError = true;
+    submitSuccess = false;
+
+    constructor(userStore) {
+        this.userStore = userStore;
+        this.balance = this.userStore.balance;
+    }
+
+    paymentDetails = async (ordernumber) => {
+        try {
+            const paymentDetails = await getPaymentDetails({
+                ordernumber,
+            });
+            this.balance = paymentDetails.balance;
+            this.amount = paymentDetails.amount;
+            this.submitError = false;
+        } catch (error) {
+            console.log(`Error in getting payment details${error}`);
```

Comment:
Log format is missing a space and colon before interpolation.

Reply:
corrected